docs/api/address/get-address-balances.schema.json

```
@@ -0,0 +1,29 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "GET request that returns address balances",
+  "title": "AccountBalance",
+  "type": "object",
+  "required": ["stx", "fungible_tokens", "non_fungible_tokens"],
+  "properties": {
+    "stx": {
+      "type": "object",
```

In a lot of the output types you'll see this prop added:

`[k: string]: unknown | undefined;`

which basically means that type can have any other property added. Often this isn't the case, we can prohibit this where possible by adding this property. Note this is required for nested objects as well.

```suggestion
      "type": "object",
      "additionalProperties": false,
```